src/Uno.UI.Runtime.Skia.Wpf/Hosting/UwpUIElementExtensions.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using windows = Windows;
+
+namespace Uno.UI.XamlHost.Skia.Wpf
+{
+	/// <summary>
+	/// Extensions for use with UWP UIElement objects wrapped by the UnoXamlHostBaseExt
+	/// </summary>
+	public static class UwpUIElementExtensions
+	{
+		private static bool IsDesktopWindowsXamlSourcePresent() => windows.Foundation.Metadata.ApiInformation.IsApiContractPresent("Windows.UI.Xaml.Hosting.HostingContract", 3);
+
+		private static windows.UI.Xaml.DependencyProperty WrapperProperty
+		{
+			get
+			{
+				if (IsDesktopWindowsXamlSourcePresent())
+				{
+					var result = windows.UI.Xaml.DependencyProperty.RegisterAttached("Wrapper", typeof(System.Windows.UIElement), typeof(UwpUIElementExtensions), new windows.UI.Xaml.PropertyMetadata(null));
+					return result;
+				}
+
+				throw new NotImplementedException();
+			}
+		}
```

Comment:
This code comes from Windows Community toolkit, but it will always evaluate `true` in Uno (as the hosting contract is always present). I will simplify the code this way